### DISCUSSION TITLE
Error while running operations

### DISCUSSION THREAD

**Discussion by deepak6422** [modin-project/modin#2771] [Q&A]
I am trying to perform an operation to find subtotal group by CustNmbr. Why I am getting this error?
![Screenshot_2021-02-23 Customer Segmentation Model - Jupyter Notebook](https://user-images.githubusercontent.com/77039531/108762526-358dac80-7576-11eb-8591-44ab07f7ff59.png)

**Comment by deepak6422** [comment]
@anmyachev Can you please help with this issue?

**Comment by devin-petersohn** [comment]
This is an output from a Ray crash. What operating system are you running on?

**Reply by deepak2431** [reply]
I am using AWS Cloud Machine. My operating system is Ubuntu 20.04 with 16 cores and 32 GB Ram.

**Reply by devin-petersohn** [reply]
This is a recent issue with Ray 1.2. I have opened an issue here to track it.

https://github.com/ray-project/ray/issues/14279

Downgrading to Ray 1.1 should solve the issue.

**Reply by deepak2431** [reply]
Thanks for your response Devin.
Can you please let me know how can I downgrade the ray version to 1.1?

**Reply by devin-petersohn** [reply]
`pip install ray==1.1`
